taichi/transforms/demote_operations.cpp

```diff
@@ -14,6 +14,30 @@ class DemoteOperations : public BasicStmtVisitor {
   DemoteOperations() : BasicStmtVisitor() {
   }
 
+  void visit(BitExtractStmt *stmt) override {
+    // @ti.func
+    // def bit_extract(input, begin, end):
+    //   return (input >> begin) & ((1 << (end - begin)) - 1)
+
+    // https://github.com/taichi-dev/taichi/pull/1795#issuecomment-682367083
+    if (stmt->get_kernel()->program.config.async_mode)
+      return;
```

Comment:
Do we still need this if we rebase to the latest commit?

Reply:
It seems to work. Thanks!